### ISSUE TITLE
Feature: how to join values across data sources, with join conditions?

### ISSUE THREAD

**Issue by bjdmeest** [kg-construct/rml-lv#16]
Came from kg-construct/rml-core#1 

it is currently possible to join values across data sources, but without join conditions (see eg https://github.com/RMLio/rml-fno-test-cases/blob/master/RMLFNOTC0009-CSV/mapping.ttl, see also https://kg-construct.slack.com/archives/C01QFSW77QF/p1615717859003600)

**Comment by elsdvlee** [comment]
Agreed with Ben to make a test case and verify if this issue can be solved using logical views.

**Comment by bjdmeest** [comment]
Fixed in https://github.com/kg-construct/rml-lv/blob/main/test-cases/RMLLVTC0003
